packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -53,7 +79,15 @@ export async function processProviderRequests(event: any) {
   return { statusCode: 200, body };
 }
 
-export async function testApi(event: any) {
+export async function testApi(event: AWSLambda.APIGatewayProxyEvent): Promise<AWSLambda.APIGatewayProxyResult> {
+  if (!event.body) {
```

Comment:
Nit: Move this to a separate validation function (as we will use for GCP as well)